src/components/UserBackgroundImage.js

```diff
@@ -1,16 +1,34 @@
-import React, { useEffect } from 'react'
+import React, { useEffect, useState } from 'react'
 import PropTypes from 'prop-types'
 import { makeStyles } from '@material-ui/core/styles'
 import { isNil } from 'lodash/lang'
-import { get } from 'lodash/object'
 import dayjs from 'dayjs'
 import isToday from 'dayjs/plugin/isToday'
 import SetBackgroundDailyImageMutation from 'src/utils/mutations/SetBackgroundDailyImageMutation'
 import { recachePage } from 'src/utils/caching'
+import { CSSTransition, TransitionGroup } from 'react-transition-group'
 
 dayjs.extend(isToday)
 const useStyles = makeStyles(() => ({
-  background: {
+  hiddenImage: {
+    visibility: 'hidden',
+  },
+  previousImage: {
+    boxShadow: 'rgba(0, 0, 0, 0.5) 0px 0px 120px inset',
+    backgroundRepeat: 'no-repeat',
+    backgroundPosition: 'center',
+    backgroundAttachment: 'fixed',
+    backgroundSize: 'cover',
+    position: 'absolute',
+    top: 0,
+    bottom: 0,
+    right: 0,
+    left: 0,
+    zIndex: 'auto',
+    backgroundImage: ({ previousImage }) =>
+      previousImage ? `url(${previousImage.imageURL})` : 'none',
+  },
```

Comment:
Move shared style into its own class so it's not duplicated and use `clsx` to apply multiple classes. Or, you can put `backgroundImage` in inline styles if you want.